### DISCUSSION TITLE
[Q&A]  glpi-agent-1.11-linux-installer.pl Installer fails every time

### DISCUSSION THREAD

**Discussion by jtfeatures** [glpi-project/glpi-agent#810] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

I ran the installer and it keeps getting stuck and I cannot figure out why from the provided verbose return (URL redacted).

```
root@server:~# perl glpi-agent-1.11-linux-installer.pl --server=https://[GLPISERVERIP.URL] --verbose
Running on linux distro: Debian GNU/Linux 12 (bookworm) : Debian GNU/Linux : 12 (bookworm)...
Installing glpi-agent v1.11...
Trying to install glpi-agent v1.11-1 on Debian GNU/Linux 12 (bookworm) release (Debian GNU/Linux:12 (bookworm))...
Extracting glpi-agent_1.11-1_all.deb ...
Trying to also install usb.ids ...
Running: apt -y install /root/glpi-agent_1.11-1_all.deb usb.ids 2>/dev/null
Failed to install glpi-agent
```

I did attempt to --force is but that has the same result.

**Comment by g-bougard** [comment]
Hi @jtfeatures 

can you try do download `glpi-agent_1.11-1_all.deb` package a run the same command but without the last output redirection which hide any error ?
```
apt -y install /root/glpi-agent_1.11-1_all.deb usb.ids
```
